packages/EasyErrorHandler/src/Exceptions/BadRequestException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Exceptions;
+
+abstract class BadRequestException extends BaseException
+{
+    /**
+     * @var int
+     */
+    protected $statusCode = 400;
+
+    /**
+     * @var string
+     */
+    protected $userMessage = 'Bad request.';
```

Comment:
I think that all default `$userMessage` should first be defined as constants so we can reuse them in the code in statements or for whatever reason, and second they should be translation keys so each app can override them this way this package could be use with another language than english out of the box.

Of course this package should provide a default translation file for english.

Reply:
I think introducing a constant would be excessive here. I don't see any reason for now. But I agree with translation keys.